File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.py
class Solution:
    def minimumLength(self, s: str) -> int:
        arr=[0]*26
        for ch in s:
            arr[ord(ch)-ord('a')] +=1
        
        for i in range(0,26):
            while arr[i]>=3:
                arr[i] -=2
        

        ans = 0
        for i in range(0,26):
            ans +=arr[i]
        
        return ans

File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        int[] arr = new int[26];
        for(char ch:s.toCharArray()){
            arr[ch-'a']++;
        }
        for(int i=0;i<26;i++){
            while(arr[i]>=3){
                arr[i] -=2;
            }
        }
        int ans = 0;
        for(int i=0;i<26;i++){
            ans +=arr[i];
        }
        return ans;
    }
}